modules/transactions.js

```diff
@@ -508,6 +508,17 @@ Transactions.prototype.undoUnconfirmed = function (transaction, cb, tx) {
 	}, tx);
 };
 
+/**
+ * Adds transaction back to transaction Pool.
+ * @param {transaction} transaction
+ * @param {function} cb - Callback function.
+ * @return {function} Calls transactionPool.queueTransaction
+ */
+Transactions.prototype.queueTransaction = function (transaction, cb) {
```

Comment:
As mentioned above, we should remove this in favour of using already defined: `Transactions.prototype.processUnconfirmedTransaction`